app/models/repository_column.rb

```diff
@@ -0,0 +1,18 @@
+class RepositoryColumn < ActiveRecord::Base
+  belongs_to :repository
+  belongs_to :team
+  belongs_to :created_by, foreign_key: :created_by_id, class_name: 'User'
+  has_many :repository_cells, dependent: :destroy
+  has_many :repository_rows, through: :repository_cells
+
+  enum data_type: Extends::REPOSITORY_DATA_TYPES
+
+  auto_strip_attributes :name, nullify: false
+  validates :name,
+            presence: true,
+            length: { maximum: Constants::NAME_MAX_LENGTH },
+            uniqueness: { scope: :team, case_sensitive: true }
```

Comment:
I think it makes sense that scope is inside repository, not team, e.g.: `uniqueness: { scope: :repository, case_sensitive: true }`.